core/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -118,10 +125,10 @@
  * {@code
  *   STORE_ROOT
  *   |- index-N           - list of all snapshot name as JSON array, N is the generation of the file
- *   |- index-latest      - contains the numeric value of the latest generation of the index file (i.e. N from above)
- *   |- snapshot-20131010 - JSON serialized Snapshot for snapshot "20131010"
+ *   |- index.latest      - contains the numeric value of the latest generation of the index file (i.e. N from above)
+ *   |- snap-20131010 - JSON serialized Snapshot for snapshot "20131010"
  *   |- meta-20131010.dat - JSON serialized MetaData for snapshot "20131010" (includes only global metadata)
- *   |- snapshot-20131011 - JSON serialized Snapshot for snapshot "20131011"
+ *   |- snap-20131011 - JSON serialized Snapshot for snapshot "20131011"
  *   |- meta-20131011.dat - JSON serialized MetaData for snapshot "20131011"
  *   .....
  *   |- indices/ - data for all indices
```

Comment:
Might be good idea to update folder name on this level as well to reflect the fact that we are using uuids now instead of actual index names.